src/Controller/UI/ReportResearchGroupDatasetStatusController.php

```diff
@@ -202,6 +202,112 @@ public function getGrpResearchGroupReport(Request $request, EntityHandler $entit
         );
     }
 
+    /**
+     * Research group dataset status report for dataset monitoring.
+     *
+     * @param Request       $request         Message information for this Request.
+     * @param EntityHandler $entityHandler   The entity handler.
+     * @param string|null   $id              Research group id.
+     *
+     * @Route(
+     *     "/report-researchgroup/grp-detail-report/{id}",
```

Comment:
maybe 'grp' is a bit confusing in this route name. This is minor as users aren't really going to see it anyway.